viewer-prototype/src/browser/trace-server-status.ts

```diff
@@ -120,11 +122,21 @@ export class TraceServerConnectionStatusContribution extends DefaultFrontendAppl
     }
 
     private async startServer() {
-        await this.traceServerConfigService.startTraceServer(this.path, this.port);
+        // TODO: Add proper error message for the port as well. Right now the server doesn't return any error code or message whenever a port is incorrect and/or the port is busy.
+        try {
+            await this.traceServerConfigService.startTraceServer(this.path, this.port);
+        } catch (err) {
+            this.messageService.error(
+                'Failed to start the trace server: no such file or directory exists. Please make sure that the path is correct in Trace viewer settings and retry');
+        }
     }
 
     private async stopServer() {
-        await this.traceServerConfigService.stopTraceServer(this.port);
+        try {
+            await this.traceServerConfigService.stopTraceServer(this.port);
+        } catch (err) {
+            this.messageService.error('Failed to stop the trace server');
```

Comment:
Might be useful to report the port, for debugging purposes. Something like:
"Failed to stop the trace server on port <port>"

Reply:
updated